.gitignore

```diff
@@ -46,3 +46,7 @@ rubocop.html
 
 # Ignore auto-generated assets
 /public/assets
+
+# Ignore passenger files
```

Comment:
Could you please ignore these files in a more general way?
For example, you could either all passenger.*.pid files or all *.pid files here.